lnwire/channel_ready.go

```diff
@@ -170,3 +171,10 @@ func (c *ChannelReady) Encode(w *bytes.Buffer, _ uint32) error {
 func (c *ChannelReady) MsgType() MessageType {
 	return MsgChannelReady
 }
+
+// String returns a human-readable description of the ChannelReady msg.
+func (c *ChannelReady) String() string {
+	return fmt.Sprintf("chan_id=%v, next_point=%x, aliasSCID=%v(uint=%d)",
+		c.ChanID, c.NextPerCommitmentPoint.SerializeCompressed(),
+		c.AliasScid, c.AliasScid.ToUint64())
+}
```

Comment:
I propose to add a unit test to have an example of such a string.

Reply:
added a property test.